### PR TITLE
bgp: beaker platform fixes #1

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
* bgp: reconnect_interval was missing from test
* bgp_af: change redist expected value to sorted
* bgp_neighbor: add log_neighbor_changes

### CODE REVIEW

File: tests/beaker_tests/cisco_bgp_af/test_bgpaf.rb [hunk 1]
```diff
@@ -208,7 +208,7 @@
   },
 }
 
-redistribute = Array[%w(static RouteMap), %w(lisp RouteMap)]
+redistribute = Array[%w(lisp RouteMap), %w(static RouteMap)]
```
Comment (robert-w-gries) [inline tests/beaker_tests/cisco_bgp_af/test_bgpaf.rb]: Why is the swap necessary?

Comment (chrisvanheuveln) [inline tests/beaker_tests/cisco_bgp_af/test_bgpaf.rb]: The redist getter return values were different on n6k (lex.sort) vs n9k (unsorted) so I added a sort to the NU getter to normalize the data.
The array above is used by puppet resource so it has to be in sorted order to match the getter output.

**Comment by robert-w-gries** [comment]
:+1:

**Comment by chburket** [comment]
:+1:

**Pull request merged**